src/config/apolloclient.ts

```diff
@@ -15,12 +16,23 @@ const gqlClient = (auth_token: string | null) => {
     };
   });
 
+  const errorLink = onError(({ graphQLErrors, networkError }) => {
```

Comment:
is this for developer debugging only? I see only errors getting printed in browser console.

Reply:
Yes, I have kept it for now.